index.html

```diff
@@ -8,7 +8,180 @@
+            <img src="./images/exhibition.png" alt="exibition" />
+            <h3>Exibition</h3>
+            <p>
+              In museums, commemorate various accomplishments of Africa's
+              artificial intelligence institution and company.
+            </p>
+          </div>
+          <div>
+            <img src="./images/forum.png" alt="Forum" />
+            <h3>Forum</h3>
+            <p>
+              Take the time to speak with professionals about a certain subject
+              and engage in discussions with them.
+            </p>
+          </div>
+          <div>
+            <img src="./images/workshop.png" alt="Workshop" />
+            <h3>Workshop</h3>
+            <p>
+              Demonstrate the utility of AI in various industries and engage in
+              detailed discussions.
+            </p>
+          </div>
+          <div>
+            <img src="./images/ignite.png" alt="AI ignite" />
+            <h3>AI connect</h3>
+            <p>
+              Get opportunities to network with and share experiences with
+              people from all over the world.
+            </p>
+          </div>
+          <button class="joinbutton" hidden>Join African AI summit 2023</button>
+        </div>
+        <a href="#" class="desktopprogram">SEE THE WHOLE PROGRAM</a>
+      </section>
+      <!--Featured speaker -->
+      <section class="speaker" id="speakers">
+        <!-- 1 -->
+        <h2 class="speakertitle">Featured Speakers</h2>
+        <hr class="outline" />
+        <div class="speakerposition">
+          <!-- all cards generated and populate data using javascript  -->
+          <div class="speakercard">
+            <img class="speakerpicture" />
+            <div>
+              <h3 class="speakername"></h3>
+              <h4 class="speakertitledetail"></h4>
+              <hr class="speakeroutline" />
+              <p class="speakerexperience"></p>
+            </div>
+          </div>
+          <div>
+            <button class="speakerbutton" id="morebutton">
+              MORE <i class="fa-solid fa-angle-down"></i>
+            </button>
+            <button class="speakerbutton" id="lessbutton" hidden>
+              LESS <i class="fa-solid fa-angle-up"></i>
+            </button>
+          </div>
+        </div>
+      </section>
+      <!-- sponsor -->
+      <section class="sponsor homesponsor" id="partners">
+        <h2>Partner</h2>
+        <hr class="partneroutline" />
+        <div class="sponsorimg">
+          <img
+            src="./images/Addis_Ababa_University_logo.png"
+            alt="addis ababa university"
+          />
+          <img
+            src="./images/EthiopianAI.png"
+            alt="Ethiopian Artificial Intelligence Institute"
+          />
+          <img src="./images/Presight_AI.png" alt="Presight AI" />
+          <img src="./images/Hawk university.png" alt="Hawk university" />
+          <img src="./images/IcogLab.png" alt="IcogLab" />
+        </div>
```

Comment:
- For semantic reasons, I suggest you use a `ul` tag to group your images and ensure all `img` tags are in a corresponding `li` tag. Please fix this 😄

Reply:
@mckent05 sponsor image were warped in `ul` and `li`